region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -447,6 +447,22 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [1, 4],
+      "name": "Speedball",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 0,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canSpeedball"
```

Comment:
Needs some sort of suitless maridia tech - it's not a perfect fit, but maybe `canWaterShineCharge` since it's obscure or `canShinechargeMovementComplex`? 

Do you have to jump in the previous room? if so, should this have an `openEnd` of 0?

Reply:
Yeah, `canWaterShineCharge` seems like a reasonable fit. There's technically no shinecharge, but it's same idea of charging from an air room into a water room. I don't think `canShinechargeMovementComplex` would work since that's more specific to moving with a shinecharge, which doesn't apply here.

You don't have to jump in the previous room, though it could often be easier to do so. Doing it in-room is probably a `canTrickyJump`, so I added that as a requirement too.